aws_ipadd

```diff
@@ -24,6 +24,7 @@ if not profile_name in config:
 config = config[profile_name]
 rule_name = config['rule_name']
 port = int(config['port'])
+protocol = config['protocol']
```

Comment:
Thanks for PR. Can you please add one below change to set default protocol as a `tcp`.
 `protocol = config['protocol'] or tcp`